probes/sastToolSnykInstalled/def.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: sastToolSnykInstalled
+short: Check that the project uses the Snyk github action
+motivation: >
+  SAST is testing run on source code before the application is run. Using SAST tools can prevent known classes of bugs from being inadvertently introduced in the codebase.
+implementation: >
+  The implementation checks whether the project invokes the snyk/actions action.
+outcome:
+  - If the project uses the snyk/actions/* action, the probe returns one finding with OutcomePositive (1).
```

Comment:
one positive outcome per matching workflow file I think?

Reply:
I don't think so, only one finding is returned?

Reply:
In terms of rawdata, we have the ability to return multiple findings. But CodeQL also just creates one finding. So the fix (if we switch) would be outside the scope of this PR.